substreams/ethereum-balancer-v3/integration_test.tycho.yaml

```diff
@@ -0,0 +1,92 @@
+substreams_yaml_path: ./substreams.yaml
+protocol_type_names:
+  - "balancer_v3_pool"
+adapter_contract: "BalancerV3SwapAdapter"
+adapter_build_signature: "constructor(address,address,address)"
+adapter_build_args: "0xbA1333333333a1BA1108E8412f11850A5C319bA9,0x136f1EFcC3f8f88516B9E94110D56FDBfB1778d1,0x000000000022D473030F116dDEE9F6B43aC78BA3"
+skip_balance_check: false 
+initialized_accounts:
+  - "0xbA1333333333a1BA1108E8412f11850A5C319bA9"
```

Comment:
Is this the vault contract? Could you please add a comment here describing why this is necessary?